CMakeLists.txt

```diff
@@ -27,6 +27,15 @@ endif()
 project(WildMeshingToolkit DESCRIPTION "A mesh optimization toolkit")
 ################################################################################
 
+if(INPUT_WMT_DATA_ROOT)
```

Comment:
Set it as cache variable with default value instead:
https://github.com/adobe/lagrange/blob/aeebd56eaaf9d3ca448174501781c8bedaa05b63/cmake/lagrange/lagrange_download_data.cmake#L16

The MAKE_DIRECTORY is also not needed (cmake will create the folder as needed with FetchContent/ExternalProject_Add).